book.toml

```diff
@@ -7,7 +7,7 @@ title = "Zcash Trailing Finality Layer"
 
 [build]
 # Follow gh pages convention; see https://nathan-at-least.github.io/auto-deploy-howto.html
```

Comment:
We should update this comment for accuracy. This is probably the best place to document the `docs -> html` symlink design, also.